gradio/demo.py

```diff
@@ -0,0 +1,273 @@
+import collections
+import os
+import tempfile
+
+from matplotlib import gridspec
+from matplotlib import pyplot as plt
+import numpy as np
+from PIL import Image
+import urllib
+
+import tensorflow as tf
+import gradio as gr
+
+from subprocess import call
+import sys
+
+import requests
+
+url1 = 'https://cdn.pixabay.com/photo/2014/09/07/21/52/city-438393_1280.jpg'
+r = requests.get(url1, allow_redirects=True)
+open("city1.jpg", 'wb').write(r.content)
+
+url2 = 'https://cdn.pixabay.com/photo/2016/02/19/11/36/canal-1209808_1280.jpg'
+r = requests.get(url2, allow_redirects=True)
+open("city2.jpg", 'wb').write(r.content)
```

Comment:
sorry for a late comment on style here: would it be possible to move these variables into main() and add a section 

if __name__ == '__main__':
  app.run(main)

This would be more consistent with other python scripts in this repo.

Reply:
is this the right section of code for this comment?

Reply:
I was referring to line 19-25 here. variables `url1` / `r` / `url2` are global variables and might hurt readability. 

deeplab2 codebase encourage removing global variables if possible, with the exception of global constants (where the variable naming would be all caps)